pkg/bff/auth/metadata.go

```diff
@@ -1,19 +1,47 @@
 package auth
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/trisacrypto/directory/pkg/bff/models/v1"
+)
 
 // AppMetadata makes it easier to serialize and deserialize JSON from the auth0
 // app_metadata assigned to the user by the BFF (and ensures the data is structured).
 type AppMetadata struct {
-	OrgID string `json:"orgid"`
-	VASPs VASPs  `json:"vasps"`
+	OrgID         string              `json:"orgid"`
+	VASPs         VASPs               `json:"vasps"`
+	Organizations map[string]struct{} `json:"organizations"`
```

Comment:
Keep in mind that this will be serialized to JSON - are you trying to ensure that `Organizations` is a `set` or just trying to implement forward compatibility if we want to store other information with the Organization? If the answer is `set` then I think it might be better to use `[]string` here and enforce the set some other way in code.

Reply:
We might want to have additional info in there at some point, however I think I agree that the `[]string` is the simplest for now.